flexget/plugins/sites/lostfilm.py

```diff
@@ -47,147 +37,128 @@ class LostFilm(object):
-                    new_title = '.'.join(
-                        [
-                            series_name_eng,
-                            episode_id,
-                            quality,
-                            'rus.LostFilm.TV',
-                            file_ext,
-                            'torrent'
-                        ]
-                    ).replace(' ', '.')
-                else:
-                    if item.get('title') is not None:
-                        new_title = '{} {}'.format(
-                            item['title'],
-                            quality)
-                    else:
-                        log.debug('Item doesn\'t have a title')
+
+                page = get_soup(response.content)
+
+                episode_id = 'S{:02d}E{:02d}'.format(season_num, episode_num)
+                for item in page.findAll('div', 'inner-box--item'):
+                    torrent_link = quality = None
+                    try:
+                        torrent_link = item.find('div', 'inner-box--link sub').a['href']  # http://tracktor.in/...
+                        quality = item.find('div', 'inner-box--label').text.strip()
+                    except:
+                        log.debug('Item doesn\'t have a link or quality')
                         continue
-                new_entry = Entry()
-                new_entry['url'] = torrent_link
-                new_entry['title'] = new_title.strip()
-                if series_name_rus:
-                    new_entry['series_name_rus'] = series_name_rus
-                if episode_name_rus:
-                    new_entry['episode_name_rus'] = episode_name_rus
-                if series_name_eng:
-                    new_entry['series_name_eng'] = series_name_eng
-                if episode_name_eng:
-                    new_entry['episode_name_eng'] = episode_name_eng
-                entries.append(new_entry)
+                    if torrent_link is None or quality is None:
+                        log.debug('Item doesn\'t have a link or quality')
+                        continue
+                    if QUALITY_MAP.get(quality):
+                        quality, file_ext = QUALITY_MAP.get(quality)
+                    else:
+                        file_ext = 'avi'
+                    if series_name_eng:
+                        new_title = '.'.join(
+                            [
+                                series_name_eng,
+                                episode_id,
+                                quality,
+                                'rus.LostFilm.TV',
+                                file_ext,
+                                'torrent'
+                            ]
+                        ).replace(' ', '.')
+                    else:
+                        if item.get('title') is not None:
+                            new_title = '{} {}'.format(
+                                item['title'],
+                                quality)
```

Comment:
I don't like these unnecessary line breaks.

Reply:
Why? Imo, it's easier to update the final string structure. However, this may be not necessary here, as hardly somebody will dislike the source torrent file name.

Reply:
Nothing is gained by these line breaks. Readability even suffers slightly imo.